source/MaterialXRuntime/RtPrim.cpp

```diff
@@ -40,6 +40,11 @@ RtRelationship RtPrim::getRelationship(const RtToken& name) const
     return rel ? rel->hnd() : RtRelationship();
 }
 
+RtConnectionIterator RtPrim::getRelationships() const
```

Comment:
Generic iteration over relationships, similar to RtPrim::getAttributes found below.

Reply:
This is a bit different compared to getAttribute/getAttributes. Since getRelationship returns a relationship object whereas getRelationships will return targets connected to the relationship objects. It would be more consistent to iterate over relationship objects, and then the caller can find the targets on each relationship.

Reply:
Sorry I misunderstood how RtConnectionIterator was used below, it's actually returning the relationship objects and not their targets. But I think that is not expected behavior for the RtConnectionIterator class, so it would be better to add a dedicated iterator for relationships instead.